src/react-components/media-tiles.js

```diff
@@ -293,12 +302,18 @@ class MediaTiles extends Component {
                 </div>
               )}
             {isHub && (
-              <div className={styles.attribution}>
-                <div className={styles.lastJoined}>
-                  <FormattedMessage key="1" id="media-browser.hub.joined-prefix" />
-                  {dayjs(entry.last_activated_at).fromNow()}
+              <>
```

Comment:
this empty tag is a new one, is this intended? i've never seen that before.

Reply:
It is a "Fragment" https://reactjs.org/docs/fragments.html